backend/ibutsu_server/constants.py

```diff
@@ -34,7 +34,12 @@
                 "description": "The field in a result to group by",
                 "type": "string",
             },
-            {"name": "sort_field", "description": "The field to sort results by", "type": "string"},
+            # Apparently the widget doesn't support this field
```

Comment:
Oh yeah, we can remove this comment now, I got rid of that field with during the postgres migration I believe.